keyboards/18bluetoothpad/rules.mk

```diff
@@ -0,0 +1,28 @@
+# MCU name

```

Comment:
Please change this file's line endings to Unix.

Reply:
Sir , i don`t understand you talk,sorry